src/libraries/System.IO.FileSystem/tests/FileStream/LockUnlock.cs

```diff
@@ -153,9 +168,7 @@ public void OverlappingRegionsFromSameProcess_AllowedOnUnix(long fileLength, lon
             }
         }
 
-        private static bool IsNotWindowsSubsystemForLinuxAndRemoteExecutorSupported => PlatformDetection.IsNotWindowsSubsystemForLinux && RemoteExecutor.IsSupported;
-
-        [ConditionalTheory(nameof(IsNotWindowsSubsystemForLinuxAndRemoteExecutorSupported))] // https://github.com/dotnet/runtime/issues/28330
```

Comment:
I tested this on WSL2 and this doesn't seem to be an issue any longer, if CI complains I will bring this condition and the link to https://github.com/dotnet/runtime/issues/28330 back.